lib/odbc_adapter/adapters/postgresql_odbc_adapter.rb

```diff
@@ -42,7 +45,7 @@ def default_sequence_name(table_name, pri_key = nil)
         "#{table_name}_#{pri_key || 'id'}_seq"
       end
 
-      def sql_for_insert(sql, pri_key, _id_value, _sequence_name, binds)
+      def sql_for_insert(sql, pri_key, binds)
```

Comment:
argument changes match psql adapter in rails 6: https://github.com/ruiinoue/rails-tutorial/blob/3c28943a63fa05ab0d064cd54101fd596c14236e/vendor/bundle/ruby/2.6.0/gems/activerecord-6.0.3.4/lib/active_record/connection_adapters/postgresql/database_statements.rb#L115